From acc6e5299c3f931fac402323d2207ddee9b838f7 Mon Sep 17 00:00:00 2001
From: Mikael Persson <mikael.persson@intel.com>
Date: Fri, 25 Oct 2013 09:33:44 +0200
Subject: Added support for flink buffer passing.

Added MEM_MODE_DRMFLINK to MemMode. This is used by USB camera HAL for buffer
sharing with video encoder.

Issue: ABRO-179
Change-Id: I1c68eff1bb2676f71e6012ae8cd60bf6bb751b5a
Signed-off-by: Mikael Persson <mikael.persson@intel.com>
---
 videoencoder/IntelMetadataBuffer.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/videoencoder/IntelMetadataBuffer.h b/videoencoder/IntelMetadataBuffer.h
index 7518d23..3db8c3a 100644
--- a/videoencoder/IntelMetadataBuffer.h
+++ b/videoencoder/IntelMetadataBuffer.h
@@ -58,6 +58,7 @@ typedef enum {
     MEM_MODE_KBUFHANDLE = 64,
     MEM_MODE_ION = 128,
     MEM_MODE_NONECACHE_USRPTR = 256,
+    MEM_MODE_DRMFLINK = 512
 }MemMode;
 
 typedef struct {
-- 
1.7.4.1

